common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgBlueprintStore.java

```diff
@@ -241,10 +242,15 @@ private void handleBlueprintImport(final File bpDir, final Map<String, VraNgBlue
 		VraNgBlueprint bp = loadBlueprintFromFilesystem(bpDir);
 		String bpID;
 
+		// Check the blueprint folder name(bpName) with the name from details.json(bp.getName())
```

Comment:
We have a practice of adding way too many inline comments and not enough documentation to the methods 😄 let's change that

Reply:
True, Not enough Doc comments. 😄
But I added this inline comment because it took me some time to identify the variables bpName and bp.getName() and understand what values they represent. With the comment, anyone can quickly see that bpName corresponds to the 'blueprint folder name' and bp.getName() corresponds to the 'name from details.json'. Idea was adding comment might make easy to understand the variables without tracing where the values are set. WDYT ? Should I remove it?

Reply:
Looks fine, keep it, but add to the documentation too